pkg/psmdb/pmm.go

```diff
@@ -340,27 +341,23 @@ func AddPMMContainer(cr *api.PerconaServerMongoDB, secret *corev1.Secret, dbPort
 	}
 
 	pmmC := PMMContainer(cr, secret, dbPort, customAdminParams)
-	if cr.CompareVersion("1.2.0") >= 0 {
-		pmmC.Resources = cr.Spec.PMM.Resources
+
+	clusterName := cr.Name
+	if len(cr.Spec.PMM.CustomClusterName) > 0 {
+		clusterName = cr.Spec.PMM.CustomClusterName
+
 	}
-	if cr.CompareVersion("1.6.0") >= 0 {
```

Comment:
How do we determine which versions to remove? Rule of the latest 3?

Reply:
yes, last 3